.travis.yml

```diff
@@ -25,3 +25,9 @@ matrix:
     - env: RUBY_VERSION=ruby-2.4 OPENSSL_VERSION=libressl-2.4
     - env: RUBY_VERSION=ruby-2.4 OPENSSL_VERSION=libressl-2.5
   allow_failures:
+    - rvm: ruby-head
+      before_install:
+        - "ruby -ropenssl -e 'puts OpenSSL::OPENSSL_VERSION'"
+        - "gem update --system"
+        - "gem update bundler"
+      script: "bundle exec rake test"
```

Comment:
This is not working (see https://travis-ci.org/ruby/openssl/builds/232773361). I suspect it's because we have 'language: c' on line 1.

We are not using Bundler so the 'before_install' and 'script' sections would be like this, I think.

~~~
before_install:
  - "rake install_dependencies"
script:
  - "rake compile -- --enable-debug"
  - "rake test"
~~~